packages/block-editor/src/store/selectors.js

```diff
@@ -314,13 +313,11 @@ export const __experimentalGetGlobalBlocksByName = createSelector(
  */
 export const getBlocksByClientId = createSelector(
 	( state, clientIds ) =>
-		map(
-			Array.isArray( clientIds ) ? clientIds : [ clientIds ],
+		( Array.isArray( clientIds ) ? clientIds : [ clientIds ] ).map(
```

Comment:
Always an array as guaranteed by the ternary.

Reply:
Offhand comment: fundamental selectors like this one really shouldn't be doing this kind of magic; they get called so often that they should be as simple and do as little work as possible (including instancing new things). Out of scope for this PR, though, your change looks good 👍